File: programmers/기본연습문제/두 정수 사이의 합.cpp
#include <string>
#include <vector>

using namespace std;
long long answer = 0;
int s, e;

long long solution(int a, int b) {
    if(a < b)
        s = a, e = b;
    else
        s = b, e = a;
    
    for(int i = s; i <= e; i++){
        answer += i;
    } 
    
    return answer;
}

File: python/기본연습문제/두 정수 사이의 합.py
def solution(a, b):
    answer = 0
    s = 0
    e = 0
    
    if a >= b:
        s = b
        e = a
    else:
        s = a
        e = b
        
    for i in range(s, e+1):
        answer += i
    
    return answer